### PR TITLE
A couple of simple edits

### PR DESCRIPTION
**Pull request by rleonid**
Update copyright notice.
Add a link to the main Trakeva module.

[Review on Reviewable](https://reviewable.io/reviews/smondet/trakeva/23)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Transactions, Keys, and Values: an attempt at a generic interface for
 transactional key-value stores with different backends.
 
 
-The interfaces are defined in the `Trakeva` module, we have, for now, one
-DB implementation of the module type `Trakeva.KEY_VALUE_STORE`:
+The interfaces are defined in the [Trakeva](src/lib/trakeva.ml) module.
```
Comment (smondet) [inline README.md]: Shouldn't this now point to the `.mli`?

File: src/lib/trakeva.ml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 (**************************************************************************)
-(*  Copyright 2014, Sebastien Mondet <seb@mondet.org>                     *)
+(*  Copyright 2015, Sebastien Mondet <seb@mondet.org>                     *)
```
Comment (smondet) [inline src/lib/trakeva.ml]: *Not-a-lawyer-but* shouldn't it be `Copyright 2014, 2015`?

File: README.md (hunk 1 above)
Comment (rleonid) [inline README.md]: True, now it should!

File: src/lib/trakeva.ml (hunk 2 above)
Comment (rleonid) [inline src/lib/trakeva.ml]: Read up on this. And it turns out that only the _original_ year is the most important. I think that updating the year is essentially just good practice.

**Pull request merged**